tests/conftest.py

```diff
@@ -1,9 +1,172 @@
 """Fixtures common across tests."""
+import pickle
+from typing import Callable, List
 
+import gym
 import pytest
 import torch
+from stable_baselines3 import PPO
+from stable_baselines3.common.policies import BasePolicy
+from stable_baselines3.common.vec_env import DummyVecEnv, VecNormalize
 
+from imitation.data import rollout
+from imitation.data.types import TrajectoryWithRew
+from imitation.data.wrappers import RolloutInfoWrapper
+from imitation.policies.base import FeedForward32Policy, NormalizeFeaturesExtractor
 from imitation.util import logger
+from imitation.util.networks import RunningNorm
+
+CARTPOLE_ENV_NAME = "CartPole-v1"
+
+
+def load_or_train_ppo(
+    cache_path: str,
+    training_function: Callable[[gym.Env], PPO],
+    venv,
+) -> PPO:
+    try:
+        return PPO.load(cache_path, venv)
+    except OSError:
+        pass
+    except AssertionError:
+        pass
+    expert = training_function(venv)
+    expert.save(cache_path)
+    return expert
+
+
+def load_or_rollout_trajectories(cache_path, policy, venv) -> List[TrajectoryWithRew]:
+    try:
+        with open(cache_path, "rb") as f:
+            return pickle.load(f)
+    except OSError:
+        pass
+    except AssertionError:
+        pass
+    rollout.rollout_and_save(
+        cache_path,
+        policy,
+        venv,
+        rollout.make_sample_until(min_timesteps=2000, min_episodes=57),
+    )
+    with open(cache_path, "rb") as f:
+        return pickle.load(f)  # TODO: not re-loading the trajectory would be nicer here
+
+
+@pytest.fixture(params=[1, 4])
```

Comment:
I think previously we only used different num envs for some of our tests (notably those in DAgger). Seems probably good to have this difference tested elsewhere, but can you quickly check what affect this has on test suite runtime? If it's a significant increase we might want to be more selective. I think we can use indirect parametrization to achieve this: https://docs.pytest.org/en/6.2.x/example/parametrize.html#indirect-parametrization